### ISSUE TITLE
[PROJECT PROPOSAL]

### ISSUE THREAD

**Issue by lcs2022026** [CognitiveLab-tech/World-of-AI#11]
## Project Request
Fruit Image Classifier

This ML model would classify a fruit image to the respective fruit class
---

| Field  |Fruit image classifier                     |
| ------ | --------------------------------- |
| About  | This ML model would classify a fruit image to the respective fruit class,and i would do this project by many classification models then i would select the best one|
| Github | @lcs2022026         |
| Email  |    lcs2022026@iiitl.ac.in           |
| Label  | Project Request                   |

https://github.com/lcs2022026

---

- [ yes] GSSOC'23 Participant
- [yes ] Contributor

**Comment by lcs2022026** [comment]
@adithya-s-k  
i am GSSOC'23 contibutor
i would like to do this project and would use search grid Cv to select the best classifier and to hypertune it

**Comment by adithya-s-k** [comment]
@lcs2022026 please follow the issue format and give me more details about the project like
- Description
- Scope
- Objectives:
- Deliverables
- Constraints
- Timeline

after you provide me with these details i will assign this issue to you

**Comment by lcs2022026** [comment]
@adithya-s-k 
description:-A fruit classification ML project is a project that uses machine learning techniques to classify fruits. [For example, one such project uses the Fruits 360 dataset and Keras to build a model that can classify between 10 different types of fruit] Another project implements an automatic classification and detection of fruits using neural network approaches for educational purposes, industries and supermarkets](https://www.hindawi.com/journals/sp/2022/4194874/) There are many other examples of fruit classification projects that use different datasets and machine learning techniques to achieve their goals.
Scope:- scope is limited to supermarkets and grocery stores 
Objectives:- To make classification of fruits in supermarkets automated ,so that it can save  a lot of time
Deliverables:- A  model would be made solving this issue
Constraints:- First I am gonna make this model on 10 fruits then i would expand it
Timeline:- I will be needing a time of 14 days , so that i can make the model and hypertune it

**Comment by adithya-s-k** [comment]
@lcs2022026 
I am thrilled to receive your project request! Your idea is truly fascinating and I am eager to see it come to life.

To ensure the project runs smoothly, please follow all guidelines provided. If you are working on an AI, ML, or DL project, we kindly ask that you create a folder for your project within the respective folders and submit your progress accordingly.
Please do follow the [project readme template
](https://github.com/CognitiveLab-tech/World-of-AI/blob/main/.github/project_readme_template.md)
To ensure that everyone has a fair chance to participate, we kindly request that you complete 75% of the work within the first week of receiving the issue, and the remaining 25% within the next 3 days(10 days in total). If for any reason, you fail to meet this deadline, we will assign the task to someone else who is equally enthusiastic about contributing to this project.

If you have any questions, feel free to contact me via email or Discord or reach out to our team of project mentors.

Thank you for your contribution and let's make this project a huge success!
